front/src/applications/stdcm/types.ts

```diff
@@ -193,7 +193,7 @@ export type StdcmLinkedPathStep = {
   uic: number;
 };
 
-export type StdcmLinkedPathResult = {
+export type StdcmLinkedTrainResult = {
   destination: StdcmLinkedPathStep;
```

Comment:
I know it's pathStep, but I think that StdcmLinkedPathStep does not really make sense... Maybe StdcmLinkedTrainExtremity would be better ?
```suggestion
  destination: StdcmLinkedTrainStep;
```